src/main/java/org/folio/edge/sip2/repositories/PatronRepository.java

```diff
@@ -149,6 +155,11 @@ public Future<PatronStatusResponse> performPatronStatusCommand(
         sessionData.isPatronPasswordVerificationRequired());
 
     return passwordVerifier.verifyPatronPassword(patronIdentifier, patronPassword, sessionData)
+      .onFailure(throwable -> {
+        if (throwable instanceof ClientException) {
+          sessionData.setErrorResponseMessage(invalidPatron(patronStatus, FALSE).result());
+        }
+      })
         .compose(verification -> {
```

Comment:
Something is funky with the indents here. Maybe the block above needs to come forward a couple of spaces to line up with eh one below.